modules/test/tls/python/src/tls_util.py

```diff
@@ -188,9 +188,10 @@ def validate_local_ca_signature(self, device_cert_path):
         # Create the file path
         root_cert_path = os.path.join(self._root_certs_dir, root_cert)
         LOGGER.info('Checking root cert: ' + str(root_cert_path))
-        args = f'{root_cert_path} {device_cert_path}'
+        args = f'"{root_cert_path}" "{device_cert_path}"'
         command = f'{bin_file} {args}'
         response = util.run_command(command)
+        LOGGER.debug(response)

```

Comment:
The response object for this method returns a tuple so doesn't lend itself to a very user friendly logging 

Error Example:
> ('ROOT: testing/unit/tls/tmp/Testrun\nDEVICE_CERT: CA', b'Error loading file testing/unit/tls/tmp/Testrun\n')

Success Example:

> ('ROOT: testing/unit/tls/tmp/Testrun CA Root.crt\nDEVICE_CERT: testing/unit/tls/certs/device_cert_local.crt\ntesting/unit/tls/certs/device_cert_local.crt: OK', b'')

Adding logging to the result should check for error before logging and only log success results since error logging is already a built in function to the underlying run_command function and would cause duplicate logs.
  ```
stdout, stderr = process.communicate()
  if process.returncode != 0 and output:
    err_msg = f'{stderr.strip()}. Code: {process.returncode}'
    LOGGER.error('Command Failed: ' + cmd)
    LOGGER.error('Error: ' + err_msg)
```

See example here of error checking of run_cmd:
https://github.com/google/testrun/blob/dev/framework/python/src/common/session.py#L143-L154

Reply:
Oh, I didn't notice that error log before.
Perhaps having the debugs logs directly into the 'run_command'  functions instead is a better idea.